client/src/main/java/org/apache/celeborn/client/read/DfsPartitionReader.java

```diff
@@ -197,7 +211,7 @@ public ByteBuf next() throws IOException, InterruptedException {
       fetchThread.submit(
           () -> {
             try {
```

Comment:
In this catch case, we could avoid to fallback to read SortedPath, [code](https://github.com/apache/celeborn/blob/79b49805e8f9b9859dcc3c5ad23a84c4b22958f8/client/src/main/java/org/apache/celeborn/client/read/DfsPartitionReader.java#L217)

Reply:
fixed, PTAL @wangshengjie123

Reply:
Hi @Z1Wu , sorry for late reply.  

LGTM, thanks